src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'book-store/src/redux/categories/CHECK_STATUS';
+
+const init = [];
+
+export default function categoriesReducer(state = init, action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return [...state, action.payload];
+    default:
+      return state;
+  }
+}
+
+export const checkStatus = (category) => ({
+  type: CHECK_STATUS,
+  payload: category,
+});
```

Comment:
- [ ] **[OPTIONAL]** I don't think we would to be dispatching a payload here, the action type should be enough.

  ```
  export const checkStatus = () => ({
    type: CHECK_STATUS,
  });
  ```